src/main/java/org/influxdb/impl/InfluxDBImpl.java

```diff
@@ -318,8 +319,9 @@ public QueryResult query(final Query query) {
       call = this.influxDBService.postQuery(this.username,
                                             this.password, query.getDatabase(), query.getCommand());
     } else {
+      String encodedCommand = UrlEscapers.urlFormParameterEscaper().escape(query.getCommand());
```

Comment:
why need this (I notice the encoded=true in all query methods)? if need it, why loss it in write method on 333 line?